Sources/TensorFlow/Layers/Sequential.swift

```diff
@@ -12,7 +12,37 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/// A layer that sequentially composes two other layers.
+/// A layer that sequentially composes two or more other layers.
+///
+/// ### Usage Example: ###
```

Comment:
For consistency, I'd recommend using `Example` instead of `Usage Example` to be more in line with non-Swift API docs. For instance: https://github.com/tensorflow/tensorflow/blob/r2.1/tensorflow/python/keras/engine/sequential.py#L43-L391. This is to do with the fact that S4TF and TF share the same search on the main .org site.

Reply:
Or maybe in your case you should use `Examples` (plural) 😊

Reply:
Good suggestions, thanks!